graphene_django/compat.py

```diff
@@ -34,11 +36,49 @@ def __init__(self, *args, **kwargs):
         ]
     ):
 
-        class ArrayField(JSONField):
+        class ArrayField(models.JSONField):
             def __init__(self, *args, **kwargs):
                 if len(args) > 0:
                     self.base_field = args[0]
                 super().__init__(**kwargs)
 
     else:
         ArrayField = MissingType
+
+
+try:
+    from django.utils.choices import normalize_choices
+except ImportError:
+
+    def normalize_choices(choices):
```

Comment:
Good find on `normalize_choices` from Django! I see it seems to have a [different signature](https://github.com/django/django/blob/fcd9d08379a2aee3b2c49eab0d0b8db6fd66d091/django/utils/choices.py#L72) and much more extensive behavior than the `normalize_choices` fallback defined here, but I suppose it's a superset of the functionality (with the additional `depth` arg being optional) and not particularly harmful to have less extensive fallback behavior, so seems alright to me.

Reply:
You are right about the different signature, but the depth parameter is used for the recursive calls from inside the function, I don't think it is useful for outside callers